packages/core/src/components/Button/Button.scss

```diff
@@ -40,7 +40,7 @@
   padding: $spacer-half $spacer-1;
 }
 
-.ds-c-button--primary {
+.ds-c-button--primary, .ds-c-button--primary:visited {
```

Comment:
Needed to add some visited styles here to prevent the buttons for getting the visited link text color

Reply:
Should this be added to all the button variations? Not just the main class, primary, and inverse? Alternatively, can we get away with just adding it to the base class `ds-c-button`? That should be on every button right?

Reply:
Oopf, I just realized something. If we want an `<a>` element to be styled as a button, we don't want it to change its styling after being clicked. That's why we add `:visited` to all these classes. This gets a bit weird when we also a transparent button variation which is styled like a link, because then you would expect it to have a `:visited` style, even though its a button. This relates to the larger "should you have links styled as buttons and vice versa" discussion.